src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -223,6 +223,7 @@ public function has($id)
             if ('service_container' === $id
                 || isset($this->aliases[$id])
                 || isset($this->services[$id])
```

Comment:
yes but: isset is faster, and isset is what we'll keep on 3.x - and this is a revert, let's top optimizing things that worked fine and had already been reviewed hundred of times.